web/packages/teleport/src/Integrations/status/AwsOidc/AwsOidcHeader.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import { Link as InternalLink } from 'react-router-dom';
+
+import { ButtonIcon, Flex, Label, Text } from 'design';
+import { ArrowLeft } from 'design/Icon';
+import { HoverTooltip } from 'shared/components/ToolTip';
+
+import cfg from 'teleport/config';
+import { AwsOidcHeaderProps } from 'teleport/Integrations/status/AwsOidc/types';
+import { getStatusAndLabel } from 'teleport/Integrations/helpers';
+
+export function AwsOidcHeader({ integration }: AwsOidcHeaderProps) {
+  if (!integration) {
+    return;
+  }
+
+  const { status, labelKind } = getStatusAndLabel(integration);
+  return (
+    <Flex alignItems="center">
+      <HoverTooltip position="bottom" tipContent="Back to Integrations">
+        <ButtonIcon
+          as={InternalLink}
+          to={cfg.routes.integrations}
+          aria-label="back"
+        >
+          <ArrowLeft size="medium" />
+        </ButtonIcon>
+      </HoverTooltip>
+      <Text bold fontSize={6} mr={2}>
+        {integration?.name}
```

Comment:
`?` is redundant, we checked if `integrations` is not null.
But I'm wondering if we shouldn't remove the check from the line 31. and let the component render with empty integration. Otherwise, the layout may jump after we fetch it (but that is only my guess, I don't know the designs).